.travis.yml

```
@@ -17,7 +17,7 @@ jobs:
     - stage: Lint
       name: whitespace
       os: linux
-      script: "[[ -z \"$(git ls-files | xargs egrep -l '\\s+$')\" ]]"
+      script: "[[ -z \"$(git ls-files | grep -v '.png$' | xargs egrep -l '\\s+$')\" ]]"
```

```suggestion
      script: "git --no-pager grep --color -line-number --full-name -I '\\s+$'"
```
@ktbarrett, thoughts?

---

When trying this locally, I need to escape the ``+`` too - what works for me is ``'\s\+$'``.

---

Let's use `-E` instead

---

Done.